apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -260,6 +260,11 @@ const EVENTS = {
 
   // Teacher Homepage
   TEACHER_HOMEPAGE_VISITED: 'Teacher Homepage Visited',
+
+  // Aichat
```

Comment:
@samantha-code - lmk if these are okay. I based constants/descriptions from the [spec](https://docs.google.com/document/d/1yIez6ooHbfwfM2W16pQvGHgKTe97AdGJN7YECTOpmBU/edit#heading=h.ghfocbs7b58g).

Reply:
Checked with Sam via [Slack](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1714748430120159) and she approved.